cmd/vulnerability-db-api-cli/design/targets.go

```diff
@@ -104,6 +104,25 @@ var _ = Service("targets", func() {
 		})
 	})
 
+	Method("Create target", func() {
+		Payload(func() {
+			Field(1, "identifier", String, "Target Identifier", func() {
+				Example("www.example.com")
+			})
+			Field(2, "tags", ArrayOf(String), "Target Tags", func() {
+				Example([]string{"sdrn:adevinta:team:security"})
```

Comment:
Just to confirm, the CreateTarget operation wont need any authorization, right?

Reply:
Exactly, endpoints that do not refer to an specific resource, like the creation endpoints (e.g.:POST /targets) do not require authorization.